### PR TITLE
Hotfix 3.10.1 

### PR DESCRIPTION
**Pull request by olsi-qose**
## Description
Orders data is sent to all stores that have a Nosto account. 

## Checklist:
- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] I have assigned the correct milestone or created one if non-existent.
- [ ] I have correctly labeled this pull request.
- [ ] I have linked the corresponding issue in this description.
- [ ] I have updated the corresponding Jira ticket.
- [ ] I have requested a review from at least 2 reviewers
- [ ] I have checked the base branch of this pull request
- [ ] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

File: classes/services/NostoOrderService.php [hunk 1]
```diff
@@ -78,6 +78,15 @@ public function sendOrder(Order $order)
             NostoHelperContext::runInContext(
                 static function () use ($order) {
                     try {
+                        //Check that the order is related to the store in context
+                        $language = Context::getContext()->language;
+                        /** @phan-suppress-next-next-line PhanUndeclaredMethod */
+                        /** @noinspection PhpUndefinedMethodInspection */
+                        if ($language instanceof LanguageCore && $order->id_lang != $language->getId()) {
```
Comment (supercid) [inline classes/services/NostoOrderService.php:85]: In fact, doesn't look like `LanguageCore` implements `getId()`. Which class are you referring to?

Comment (olsi-qose) [reply classes/services/NostoOrderService.php:85]: It does. Here's [the line](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.6/classes/Language.php#L1745). 
But `Context::getContext()->language` might also return an object of type `PrestashopBundle\Install\Language` which does not have the method.

Comment (supercid) [reply classes/services/NostoOrderService.php:85]: That's not available prior to >= 1.7.6.9.
Maybe use reflection to get the ID?

Comment (supercid) [reply classes/services/NostoOrderService.php:85]: Actually is public, so ->id should be fine
https://github.com/PrestaShop/PrestaShop/blob/1.6.1.16/classes/Language.php#L29

**Review by supercid — APPROVED** [review]

**Comment by supercid** [comment]
Check the base branch if you're making a hotfix. Or make a full release already with what we already have on develop

**Pull request merged**